utils/windowfunction/wf_percentile.cpp

```diff
@@ -89,31 +91,19 @@ boost::shared_ptr<WindowFunctionType> WF_percentile<T>::makeFunction(int id, con
             }
 
             case CalpontSystemCatalog::DECIMAL:
+            case CalpontSystemCatalog::UDECIMAL:
             {
-                if (wc->functionParms()[0]->resultType().colWidth < 16)
+                if (wc->resultType().colWidth < datatypes::MAXDECIMALWIDTH)
```

Comment:
For a lot of other window functions, we are still using `wc->functionParms()[0]->resultType().colWidth`, and not `wc->resultType().colWidth`.

Is this deviation intentional for wf_percentile.cpp?

Reply:
Yes. Percentile has a decimal literal as an argument.